docs/reference/commandline/events.md

```diff
@@ -51,6 +51,7 @@ The currently supported filters are:
 * container
 * event
 * image
+* labels
```

Comment:
Hm, I think this is the first filter we have that has a *key* and *value*, or is that not supported? Can I, for example, do:
`--filter label=foo` to show al events coming from a container/image with a label `foo`, irregardless of its value?
Can I also filter on the *value* of that label? `--filter label=foo=bar`

Because this is new here, I think we need a small example, and some change in the wording above